From 0bd668ab4bfa8387ba230e0db2d5d2dd2cab73c6 Mon Sep 17 00:00:00 2001
From: hoahuynh <hoa.huynh.ud@renesas.com>
Date: Wed, 15 Apr 2020 00:46:15 +0700
Subject: [PATCH 250/441] arm64: dts: renesas: r8a774e1: Add USB3.0 host device
 node

This patch adds USB3.0 (xHCI) host device node for SoC R8A774E1

Signed-off-by: hoahuynh <hoa.huynh.ud@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1.dtsi | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
index c2e03eb..d16cd54 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
@@ -1632,8 +1632,14 @@
 		};
 
 		xhci0: usb@ee000000 {
+			compatible = "renesas,xhci-r8a774e1",
+				"renesas,rcar-gen3-xhci";
 			reg = <0 0xee000000 0 0xc00>;
-			/* placeholder */
+			interrupts = <GIC_SPI 102 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 328>;
+			power-domains = <&sysc R8A774E1_PD_ALWAYS_ON>;
+			resets = <&cpg 328>;
+			status = "disabled";
 		};
 
 		usb3_peri0: usb@ee020000 {
-- 
2.7.4

